pages_deployment.go

```diff
@@ -181,6 +210,32 @@ func (api *API) GetPagesDeploymentStageLogs(ctx context.Context, params GetPages
 	return r.Result, nil
 }
 
+// GetPagesDeploymentStageLogs returns the logs for a Pages deployment stage.
+//
+// API reference: https://api.cloudflare.com/#pages-deployment-get-deployment-stage-logs
+func (api *API) GetPagesDeploymentLogs(ctx context.Context, params GetPagesDeploymentLogsParams) (PagesDeploymentLogs, error) {
+	v := url.Values{}
+	if params.Size > 0 {
+		v.Set("size", strconv.Itoa(params.Size))
+	}
+
+	uri := fmt.Sprintf("/accounts/%s/pages/projects/%s/deployments/%s/history/logs", params.AccountID, params.ProjectName, params.DeploymentID)
```

Comment:
you'll want to check that `params.AccountID`, `params.ProjectName`, `params.DeploymentID` are not empty before trying to use them here.